README.md

```diff
@@ -80,6 +80,14 @@ To see recommended feeder promotions, run:
 $ hack/stabilization-change.py
 ```
 
+##### Promoting edges from previous minor to latest minor version in stable channel
+
+To promote edges from the previous minor version to the latest minor version in the stable channel, run the following script at least 3 times when it is time and then send the pull request.
```

Comment:
We could be fancy and find the transitive closure (or build this into the script entirely):

```
while ./hack/release-stable-minor.sh 4.12 && ! git diff --quiet ./channels ./internal-channels/; do
  git add ./channels/ ./internal-channels/
done
```

Reply:
+1 from me to teaching the robots to do the tedious things, so we don't have to rely on docs->human to get those steps across.